modules/release-notes/ref-release-notes-deprecated-functionalities.adoc

```diff
@@ -4,6 +4,51 @@
 
 This section lists deprecated functionalities in {product} {product-version}.
 
+[id="deprecated-functionality-rhidp-5350"]
+== Deprecation of `podman` and `ImageStreams` approach for Helm chart mirroring
 
-None.
+The previous method using `podman` and `ImageStreams` for Helm chart mirroring has been deprecated. You are now required to transition to `oc-mirror` for Helm chart and related container image mirroring. This change enables support for fully disconnected and partially disconnected environments, ensuring a {company name}-supported airgapping process.
```

Comment:
```suggestion
The previous method using `podman` and `ImageStreams` for Helm chart mirroring has been deprecated. You are now required to transition to `oc-mirror` for Helm chart and related container image mirroring on OCP. This change enables support for fully disconnected and partially disconnected environments, ensuring a {company name}-supported airgapping process.
```

Reply:
Also, `{company-name}` does not seem to render correctly in the preview:

![image](https://github.com/user-attachments/assets/7877acad-0530-40ba-82b9-91cf1d5320a7)

Reply:
Fixed it